wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -0,0 +1,782 @@
+- Aktueller Skin: `Standard` (erste Option)
+
+*Hinweis für Übersetzer: Wenn du einen Artikel mit Screenshots des Spiels übersetzt, dann darf die Sprache des Spiel-Clients in der übersetzten Sprache sein.*
+
+### Bilderlinks
+
+Bilder dürfen nicht Teil eines Linktextes sein.
+
+Flaggensymbole neben Links zu Benutzern müssen separat vom Linktext sein. Siehe folgendes Beispiel:
+
+```markdown
+![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
+```
+
+### Flaggensymbole
+
+*Für eine Liste der Flaggensymbole, siehe: [Issue \#328](https://github.com/ppy/osu-wiki/issues/328)*
+
+Die Flaggensymbole benutzen eine Zwei-Buchstaben-Kodierung (alle in Großschreibweise) und enden in `.gif`. Benutze beim Hinzufügen eines Symbols in der Linie dieses Format:
+
+```markdown
+![](/wiki/shared/flag/xx.gif)
+```
+
+Wobei `xx` der [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) zwei-Buchstaben Ländercode für die Flagge ist.
+
+Der volle Ländername sollte in den Titeltext eingefügt werden. Der Ländercode im alternativen Text ist optional, aber muss bei allen Flaggen im Artikel angewendet werden.
+
+## Tabellen
+
+Tabellen auf der Webseite unterstützen Titel nur in der ersten Zeile.
+
+Tabellen müssen nicht schön gemacht werden (füge keine extra Leerzeichen ein, um die Abstände gleichmäßig zu machen). Sie müssen einen senkrechten Strich (`|`) auf der linken sowie rechten Seite haben und jede Zelle muss mit einem Leerzeichen auf jeder Seite versehen sein. Leere Zellen müssen einen senkrechten Strich, gefolgt von zwei Leerzeichen, gefolgt von einem weiteren senkrechten Strich haben.
+
+Die Trennzeichen-Zeile (die nächste Zeile nach den Spaltenüberschriften) darf nur drei Zeichen pro Spalte benutzen (und mit einem Leerzeichen auf jeder Seite versehen sein), die wie folgt aussehen müssen:
+
+- `:--` (linksbüngig)
+- `:-:` (mittig ausrichten)
+- `--:` (rechtsbündig)
+
+---
+
+Das folgende ist ein Beispiel, wie eine Tabelle aussehen sollte:
+
+```markdown
+| Team "Picturesque" Rot | Punktestand | Team "Statuesque" Blau | Durchschnittliche Beatmap-Sterne |
+| :-- | :-: | --: | :-- |
+| **peppy** | 5 - 2 | pippi | 9.3 Sterne |
+| Aiko | 1 - 6 | **Alisa** | 4.2 Sterne |
+| Ryūta | 3 - 4 | **Yuzu** | 5.1 Sterne |
+| **Taikonator** | 7 - 0 | Tama | 13.37 Sterne |
+| Maria | Kein ausgetragenes Spiel | Mocha |  |
+```
+
+## Blockzitate
+
+Blockzitate sind auf das Zitieren von Texten von anderen beschränkt. Sie dürfen nicht dazu benutzt werden, um anderen Text zu formatieren.
+
+## Thematische Brüche
+
+Thematische Brüche (auch bekannt als horizontale Linie) sollte nur selten verwendet werden. Ein paar sinnvolle Benutzungen sind hier aufgelistet (aber nicht darauf beschränkt):
+
+- Bilder von Text trennen
+- Mehrere Bilder, die direkt hintereinander sind, voneinander trennen
+- Das Thema innerhalb eines Abschnitts wechseln
+
+Diese müssen eine leere Zeile vor und nach dem Markup haben. Thematische Brüche dürfen nur drei Bindestriche nutzen, wie unten gezeigt:
```

Comment:
```suggestion
Diese müssen eine leere Zeile vor und nach dem Markup haben. Thematische Trennungen dürfen nur drei Bindestriche nutzen, wie unten gezeigt:
```
`Unterbrechung` or `Umbruch` might work too (also on line 772), but `Bruch` sounds wrong to me.